src/Components/Wishlist/Wishlist.css

```diff
@@ -0,0 +1,93 @@
+@import url("../../style.css");
+
+.product-container{
+    height: 34rem;
+    width: 79.8rem;
+    display: flex;
+    justify-content: center;
+    margin-top: 1rem;
+}
+.section-first{
+    height: 25rem;
+    width: 60rem;
+    display: flex;
+    margin-top: 5rem;
```

Comment:
instead of this, you can give values like margin: 5rem 0 0 -5rem;
(margin: top right bottom left)